ipynb/balance_trees.ipynb

```diff
@@ -0,0 +1,488 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# __Balance Trees__"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "\n",
+    "We will introduce the concept of balances here.  Balances were originally introduced in the geological sciences, and have come under several names, such as sequential binary partitions, or CoDa dendrograms.  Here we will refer to the visualizations representing balances as Balance Trees.\n",
+    "\n",
+    "Balances are an alternative way to interpret relative abundance information, without worrying about dependency properties between features inherient in multinomial distributions.  Often times, we cannot count all of the individuals within a community, and we are stuck analyzing representative samples.  If we have a community of 2 species, we can visualize the ratio or *balance* between these two species.Below is a more intuitive explanation of a balance.\n",
+    "\n",
+    "<img src=\"images/Slide1.jpg\">\n",
+    "\n",
+    "Here we have two species, Red and Green that have equal counts.  The balance is taken as the log ratio between them.  In our convention, we will always take the ratio of the left species over the right species.  So in this case the balance is calculated as\n",
+    "\n",
+    "$\\ln \\frac{Red}{Green} = \\ln \\frac{5}{5} = \\ln 1 = 0$\n",
+    "\n",
+    "If the number of red individuals outnumber the number of green individuals 5 to 1, you'll end up with a positive balance of\n",
+    "\n",
+    "$\\ln \\frac{Red}{Green} = \\ln \\frac{5}{1} = \\ln 5$\n",
+    "\n",
```

Comment:
I double checked.  Those formulas should be ok.  Just changed all of the lns to logs